keras_nlp/samplers/beam_sampler_test.py

```diff
@@ -31,36 +34,37 @@ def setUp(self):
         self.vocab_size = len(self.int_lookup)
 
         def next(prompt, cache, index):
-            batch_size = tf.shape(prompt)[0]
+            batch_size = ops.shape(prompt)[0]
```

Comment:
Not related to this PR, now I am starting to feel that it would be harder for contributors to write PRs because our operations comes from `ops` instead of the usual libraries like `tf` or `torch`. 
If things don't work out at ops level, contributors will need to write the ops correctly.

Reply:
Agreed! Something to work out over the long term.

Reply:
This is an important point: Keras Core development is more expensive than tf.keras! Everything is a tradeoff for sure